services/src/main/scala/cromwell/pipeline/service/ProjectService.scala

```diff
@@ -4,26 +4,32 @@ import java.util.UUID
 
 import cromwell.pipeline.datastorage.dao.repository.ProjectRepository
 import cromwell.pipeline.datastorage.dto._
-import cromwell.pipeline.service.Exceptions.{ ProjectAccessDeniedException, ProjectNotFoundException }
 import cromwell.pipeline.model.wrapper.UserId
+import cromwell.pipeline.service.Exceptions.{
+  ProjectAccessDeniedException,
+  ProjectIllegalStateException,
+  ProjectNotFoundException
+}
 
 import scala.concurrent.{ ExecutionContext, Future }
+import scala.util.{ Failure, Success, Try }
 
 class ProjectService(projectRepository: ProjectRepository, projectVersioning: ProjectVersioning[VersioningException])(
   implicit executionContext: ExecutionContext
 ) {
 
-  def getProjectById(projectId: ProjectId): Future[Option[Project]] =
-    projectRepository.getProjectById(projectId)
+  private[service] def getProjectById(projectId: ProjectId, userId: UserId): Future[Project] =
```

Comment:
I suggest to rename this to getUserProjectById

Reply:
Done